examples/angularjs_require/test/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "private": true,
+  "devDependencies": {
+    "grunt": "0.4.5",
+    "grunt-contrib-jasmine": "0.8.1",
+    "grunt-template-jasmine-requirejs": "0.2.0",
```

Comment:
I am always fine with locking things unless you have infinite trust in the future maintainers of these libs to follow semver.